testing/README.md

```diff
@@ -10,13 +10,18 @@ Checkout `ivadomed/.github/workflows/run_tests.yml` to see how tests are run on
 
 ## Running Locally
 
-1. Download the required dataset(s) using the `ivadomed` command line tools:
+1. Install dependencies
+```
+pip install pytest pytest_console_scripts
+```
+
+2. Download the required dataset(s) using the `ivadomed` command line tools:
```

Comment:
I just testing the other day and I didn't have to do this. The tests are set up to auto-download these datasets (which is genius!)